### PR TITLE
Initial colima environment

### PR DESCRIPTION
**Pull request by rossturk**
https://github.com/abiosoft/colima

### CODE REVIEW

File: colima/test.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return 1
+  fi
+  echo ">>> '$1' command exists"
+}
+
+docker_is_running() {
+  local elapsed=0 
+
+  while [ 0 -lt 45 ]; do
+      if docker info &> /dev/null; then
```
Comment (rossturk) [inline colima/test.sh:17]: @garbas when I watched this run in CI it came back *way* too quickly. Does our CI runner already have a Docker daemon running?

Comment (garbas) [reply colima/test.sh:17]: yes, there is docker running on CI workers.

Comment (garbas) [reply colima/test.sh:17]: Is there a way to run colima alongside docker?

Comment (garbas) [reply colima/test.sh:17]: probably something to do with setting `DOCKER_HOST` variable

Comment (rossturk) [reply colima/test.sh:17]: for now I have disabled this part of the test - we can add it back later?

**Review by barstoolbluz — APPROVED** [review]

**Pull request merged**